db/database.go

```diff
@@ -258,7 +258,10 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 
 	}
 
-	go context.watchDocChanges()
+	if !context.UseXattrs() {
+		go context.watchDocChanges()
+	}
+
```

Comment:
Can you add comment like "Prevent bucket shadowing code to run when using Xattrs, since they are incompatible"?

Also, what about instead / additionally updating `context.watchDocChanges()` to have a second check?

```
func (c *DatabaseContext) watchDocChanges() {
	if context.UseXattrs() {
                // Prevent bucket shadowing code to run when using Xattrs, since they are incompatible
		return
	}

```

That seems safer in case other parts of the code also call this method.